src/app/todo-list/todo-list.component.html

```diff
@@ -0,0 +1,64 @@
+<div class="todo-container">
+  <div *ngIf="!todos?.length">
+    Your Todo List is Empty!
+  </div>
+
+  <ul class="list-group" *ngIf="todos?.length">
+    <li class="list-group-item todo-item" *ngFor="let todo of todos">
+      <div class="row">
+        <div class="col-xs-12 d-flex justify-content-between">
+          <ng-container *ngIf="todo.id !== editingTodoId; else editMode">
+            <span [class.completed]="todo.status === 'done'">{{this.todo.title}}</span>
+          </ng-container>
+          <ng-template #editMode>
+            <ng-container *ngIf="todo.id === editingTodoId">
+              <input [(ngModel)]="todo.title">
+            </ng-container>
+          </ng-template>
+          <span [class.green]="todo.status === 'done'">
+            status: {{this.todo.status}}
+          </span>
+        </div>
+      </div>
+      <hr>
+      <div class="row">
+        <div class="col-xs-12 d-flex justify-content-between">
+          <button
+            type="button"
+            class="btn btn-success btn-sm"
+            (click)="completeTodo(this.todo.id)"
+            [disabled]="this.todo.status === 'done'"
```

Comment:
I'll just say the same what Pavel said - here you should also replace 'done' for an enum that you will use in your .ts file